misk/src/main/kotlin/misk/web/DispatchMechanism.kt

```diff
@@ -8,8 +8,9 @@ import org.eclipse.jetty.http.HttpMethod
  */
 enum class DispatchMechanism {
```

Comment:
Grouping HTTP methods together in this enum was done as a separate commit. Trivial to drop if we feel strongly about alphabetical order.